packages/wallets/src/features/cashier/modules/Transfer/components/TransferMessages/TransferMessages.tsx

```diff
@@ -13,7 +13,7 @@ const TransferMessages = () => {
     return (
         <FadedAnimatedList className='wallets-transfer-messages'>
             {messages.map(message => (
-                <WalletAlertMessage key={message.text + message.type} message={message.text} type={message.type} />
+                <WalletAlertMessage key={message.type} message={message.text} type={message.type} />
```

Comment:
```suggestion
                <WalletAlertMessage key={message.type} message={message.text} type={message.type} />
```
why is this change made? This makes the key less specific and could cause issue.

Reply:
Yes @yashim-deriv, 
I do understand but if we pass the text as the key and the user changes input or the exchange_rate is updated, the entire message re-renders as a fresh node and it causes the animation to re-trigger. 
This does not feel right as the animation will continuously happen when the user is giving the value.

I am not sure about how can I make it unique without removing the text 😕

Reply:
ah i get it now. if that is the case just do `wallet_alert_message_${index}-${message.type}`